server.js

```diff
@@ -159,8 +158,8 @@ function Weather(weatherObject) {
 
 function Park(parkObject) {
   this.name = parkObject.name;
-  this.address = parkObject.address;
-  this.fee = parkObject.fee;
+  this.address = parkObject.addresses[0];
+  this.fee = parkObject.entrancefee[0].cost;
   this.description = parkObject.description;
```

Comment:
on 170/171, from today's video, a .then seems a great suggestion to add for future server functionality--